src/pages/gen2/build-a-backend/data/connect-to-API/index.mdx

```diff
@@ -13,206 +13,295 @@ export function getStaticProps(context) {
 
 }
 
-# Connect app code to API
+In this guide, you will connect your application code to the backend API using the Amplify Libraries. We will review how to install and configure Amplify Libraries.
 
-You can use the Amplify library to connect GraphQL APIs created in various ways:
-- Amplify vNext-generated API (backend/data.ts)
-- Amplify Classic-generated API (`backend/api/<api-name\>/schema.graphql`)
-- any existing AWS AppSync GraphQL API
-- GraphQL APIs hosted using API Gateway
+Before you begin, you will need:
+- your cloud sandbox with an Amplify Data resource up and running (`npx amplify sandbox`)
+- A frontend application set up. For instructions on setting up an app, see [Project Setup: Create your application](https://docs.amplify.aws/lib/project-setup/create-application/q/platform/js/)
```

Comment:
@renebrandel general comment that links on this page should be updated to new pages - would project set up here go to a Gen2 get started?